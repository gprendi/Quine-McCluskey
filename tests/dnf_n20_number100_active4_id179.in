d 20
00000000000000000000
00000000000100000000
00100100000000100000
00000100000000100000
00100100000000000000
00100000000000100000
00000100000100100000
00000000000100100000
00000000000000100000
00100000000100100000
00000100000000000000
00000100000100000000
00100000000100000000
00100100000100100000
00100000000000000000
00100100000100000000
