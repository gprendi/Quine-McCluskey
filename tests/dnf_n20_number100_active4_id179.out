p dnf 20 0
-1 -2 -4 -5 -7 -8 -9 -10 -11 -13 -14 -16 -17 -18 -19 -20 0
